src/tests/unit/tests/views/content/__snapshots__/markup.test.tsx.snap

```diff
@@ -1,367 +1,472 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ContentPage .Markup <Column> renders 1`] = `
-<div
-  className="column"
->
-  INSIDE COLUMN
-</div>
+<DocumentFragment>
+  <div
+    class="column"
+  >
+    INSIDE COLUMN
+  </div>
+</DocumentFragment>
 `;
 
 exports[`ContentPage .Markup <Columns> renders 1`] = `
-<div
-  className="columns"
->
-  INSIDE COLUMNS
-</div>
-`;
-
-exports[`ContentPage .Markup <Do> renders 1`] = `
-<Column>
+<DocumentFragment>
   <div
-    className="do-header"
+    class="columns"
   >
-    <h2>
-      Do
-    </h2>
-    <CheckIcon />
+    INSIDE COLUMNS
   </div>
+</DocumentFragment>
+`;
+
+exports[`ContentPage .Markup <Do> renders 1`] = `
+<DocumentFragment>
   <div
-    className="do-section"
+    class="column"
   >
-    THINGS TO DO
+    <div
+      class="do-header"
+    >
+      <h2>
+        Do
+      </h2>
+      <mock-checkicon />
+    </div>
+    <div
+      class="do-section"
+    >
+      THINGS TO DO
+    </div>
   </div>
-</Column>
+</DocumentFragment>
 `;
 
 exports[`ContentPage .Markup <Dont> renders 1`] = `
-<Column>
+<DocumentFragment>
   <div
-    className="dont-header"
+    class="column"
   >
-    <h2>
-      Don't
-    </h2>
-    <CrossIcon />
-  </div>
-  <div
-    className="dont-section"
-  >
-    DON'T DO THIS
+    <div
+      class="dont-header"
+    >
+      <h2>
+        Don't
+      </h2>
+      <mock-crossicon />
+    </div>
+    <div
+      class="dont-section"
+    >
+      DON'T DO THIS
+    </div>
   </div>
-</Column>
+</DocumentFragment>
 `;
 
 exports[`ContentPage .Markup <Fail> renders 1`] = `
-<Column>
-  <div
-    className="fail-header"
-  >
-    <CrossIcon />
-     
-    <h3>
-      Fail
-    </h3>
-  </div>
+<DocumentFragment>
   <div
-    className="fail-section"
+    class="column"
   >
-    I FAILED :(
+    <div
+      class="fail-header"
+    >
+      <mock-crossicon />
+       
+      <h3>
+        Fail
+      </h3>
+    </div>
+    <div
+      class="fail-section"
+    >
+      I FAILED :(
+    </div>
   </div>
-</Column>
+</DocumentFragment>
 `;
 
 exports[`ContentPage .Markup <Highlight> renders 1`] = `
-<span
-  className="highlight"
->
-  HIGHLIGHTED
-</span>
+<DocumentFragment>
+  <span
+    class="highlight"
+  >
+    HIGHLIGHTED
+  </span>
+</DocumentFragment>
 `;
 
-exports[`ContentPage .Markup <HyperLink> renders 1`] = `
-<NewTabLink
-  href="http://my.link"
-  onClick={[Function]}
->
-  LINK TEXT
-</NewTabLink>
-`;
+exports[`ContentPage .Markup <HyperLink> renders 1`] = `<DocumentFragment />`;
```

Comment:
NewTabLink should be seen inside the document fragment, please compare the old and new snaphsots

Reply:
I have already used  expectMockedComponentPropsToMatchSnapshots([CheckIcon, CrossIcon, NewTabLink, Helmet]); Not sure what else needs to be added.